test/envoye2e/driver/fake_stackdriver.go

```diff
@@ -133,11 +153,24 @@ func (s *LoggingServer) ListMonitoredResourceDescriptors(
 func (e *MeshEdgesServiceServer) ReportTrafficAssertions(
 	ctx context.Context, req *edgespb.ReportTrafficAssertionsRequest) (
 	*edgespb.ReportTrafficAssertionsResponse, error) {
+	log.Printf("receive ReportTrafficAssertionsRequest %+v", *req)
```

Comment:
nit: Is this log intentionally there?

Reply:
Yeah I think it does not harm to just log every received request in the fake server for easier debug.